support-frontend/assets/pages/supporter-plus-landing/setup/legacyActionCreators.ts

```diff
@@ -0,0 +1,583 @@
+// ----- Imports ----- //
```

Comment:
This is [contributionsLandingActions.ts](https://github.com/guardian/support-frontend/blob/9d8786c148917d6a399dd5c4a6ccd7ea1b960531/support-frontend/assets/pages/contributions-landing/contributionsLandingActions.ts) with unused action creators removed. Everything that remains is related to payment and form submission, which will need some focused work in the new year.